osx/scripts/postinstall-launchd-jenkins

```diff
@@ -44,7 +44,13 @@ else
     dscl . -append /Groups/jenkins GroupMembership jenkins
 fi
 
-find "$JENKINS_HOMEDIR" \( -not -user jenkins -or -not -group jenkins \) -print0 | xargs -0 chown jenkins:jenkins
+# identify the real default group name for user jenkins
+groupid=`dscl . read /Users/jenkins PrimaryGroupID | awk '{print $2}'`
+gname=`id -n -g $groupid`
+
+echo "Using jenkins:${gname} as file owner and group for jenkins daemon files"
+
+find "$JENKINS_HOMEDIR" \( -not -user jenkins -or -not -group ${gname} \) -print0 | xargs -0 chown jenkins:${gname}
```

Comment:
Not really changed in this PR, but isn't this as horribly slow as the RPM installer's [JENKINS-23273](https://issues.jenkins-ci.org/browse/JENKINS-23273)?